build/Toolset/Toolset.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
```

Comment:
VS 2017 seems to allow it at the very least and it preserves the comments (LUT and several Roslyn projects were already doing this without issue).